hirte/test/ini/config_test.c

```diff
@@ -0,0 +1,28 @@
+#include "../../src/ini/config.h"
+#include "../../src/common/hashmap/hashmap.h" 
+#include "../../src/common/parse-util.c"
+#include <stdlib.h>
+#include <stdio.h>
+
+#define assert_se(expr)          \
+        do {                     \
+                if (!(expr)) {   \
+                        exit(1); \
+                }                \
+        } while (false)
+
+bool test_config() {
+    printf("Starting the parse test\n");
+    char * config_file = "../../test/ini/example.ini";
```

Comment:
We probably want to have more test cases than only one file. For example, how does our code work when the file is empty? What happens when one IP in the IP list is erroneous (e.g. 127.0.1)? 
Please create another directory `test/ini/data/` and move the `example.ini` there. In this directory we can bundle all test cases.